packages/rum/src/boot/rum.spec.ts

```diff
@@ -10,7 +10,7 @@ import {
   SESSION_KEEP_ALIVE_INTERVAL,
   THROTTLE_VIEW_UPDATE_PERIOD,
   View,
-} from '../domain/rumEventsCollection/view/viewCollection'
+} from '../domain/rumEventsCollection/view/trackViews'
```

Comment:
This naming is not really satisfying since trackXxx and xxxCollection means pretty much the same thing 😕 
I'd be willing to go with that for now and wait the end of the migration to have a better overview and see what we can do.